MainPanel.c

```diff
@@ -30,25 +30,25 @@ void MainPanel_updateLabels(MainPanel* this, bool list, bool filter) {
    FunctionBar_setLabel(bar, KEY_F(4), filter ? "FILTER" : "Filter");
 }
 
-static void MainPanel_pidSearch(MainPanel* this, int ch) {
+static void MainPanel_idSearch(MainPanel* this, int ch) {
    Panel* super = (Panel*) this;
-   pid_t pid = ch - 48 + this->pidSearch;
+   int id = ch - 48 + this->idSearch;
```

Comment:
Integer truncation compared to previously on systems with pid_t being 64-bit.

Reply:
Hmm?  There are no systems with 64 bit PIDs, are there?

Reply:
OK, I'll go back to using pid_t here.